pkgs/by-name/li/libacars/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  cmake,
+}:
+stdenv.mkDerivation rec {
+  pname = "libacars";
+  version = "2.2.0";
+
+  src = fetchFromGitHub {
+    owner = "szpajder";
+    repo = "libacars";
+    rev = "v${version}";
+    hash = "sha256-2n1tuKti8Zn5UzQHmRdvW5Q+x4CXS9QuPHFQ+DFriiE=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  cmakeFlags = [
+    "-DCMAKE_INSTALL_LIBDIR=lib"
+  ];
+
+  meta = with lib; {
+    homepage = "https://github.com/szpajder/libacars";
+    description = ''
```

Comment:
Please make sure the description follows https://nixos.org/manual/nixpkgs/unstable/#var-meta-description